### DISCUSSION TITLE
Simple Checkbox Filter

### DISCUSSION THREAD

**Discussion by Ref-Bit** [TanStack/table#2812] [Q&A]
Hey, I've been trying to implement a simple checkbox filter. The function I made is like so
`
const handleChecked = (e) => {
  if(e.target.checked){
    const filtered = preFilteredRows.filter((row) => row.values[id] !== "-")
    } 
  }
});`

Where **filtered** is the output when the checkbox is true. I'm having a hard time integrating this simple function as a custom filter type in react-table. Any help is appreciated.

**Comment by gargroh** [comment]
Code sandbox url of the same please

**Reply by Ref-Bit** [reply]
Here's an example of what I want to achieve using react-table package 

[Code Sandbox](https://codesandbox.io/s/simple-checkbox-filter-t8njz?file=/src/App.js)
